### DISCUSSION TITLE
Dashboard variables aren't supposed to be scrolled out of view

### DISCUSSION THREAD

**Discussion by gergaly** [grafana/grafana#62707] [Dashboards, Panels & Visualizations]
I'm using grafana dashboards with variables. On my screen I see 3 graph panels. To see more I have to scroll down. But when I scroll down the variable drop down boxes scroll out of view. So to change a variable I have to scroll back up, change it and then again scroll down again. I think the variables should behave like the timepicker, they should be always visible.

Grafana Open Source, v9.3.2 (21c1d14e91), run in docker.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424).

This and more, are being discussed in https://github.com/grafana/grafana/issues/16452
